File: beginner/typescript/capitalize.ts
function capitalize(str: string): string {
  const result: string[] = [];
  for (let _str of str.split(" ")) {
    const modified_string = _str[0].toUpperCase().concat(_str.slice(1));
    result.push(modified_string);
  }
  return result.join(" ");
}

const str1 = "a short sentence";
const str2 = "a lazy fox";
const str3 = "look, it is working";

console.log(capitalize(str3));

File: beginners/capitalize/capitalize.py
def capitalize(str):
  caps_strs = [x.capitalize() for x in str.split(' ')]
  return ' '.join(caps_strs)

str1 = 'a short sentence'
str2 = 'a lazy fox'
str3 = 'look, it is working'

print(capitalize(str3))
